scopesim/effects/psf_utils.py

```diff
@@ -258,12 +258,14 @@ def rotational_blur(image, angle):
     """
     image_rot = np.copy(image)
 
-    n_angles = 0
     rad_to_deg = 57.29578
     edge_pixel_unit_angle = np.arctan2(1, (image.shape[0] // 2)) * rad_to_deg
+    n_angles = 1
+    add_angles = 1
     while abs(angle) > edge_pixel_unit_angle and n_angles < 25:
         angle /= 2.
         image_rot += spi.rotate(image_rot, angle, reshape=False, order=1)
-        n_angles += 1
+        n_angles += add_angles
```

Comment:
I've integrated a simplified version of this into the dev_master version directly,

    n_angles *= 2

as it does the same as what you are proposing here, assuming initial values of `n_angles=1` and a`dd_angles=1`